### PR TITLE
create a c-sharp content entry for math funchtion min.

### LINKED ISSUE #1610: [Term Entry] C# Math Functions `.Min()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `Math.Min()` method.

The entry should go in a new file in `docs/content/c-sharp/concepts/math-functions/terms/min/min.md`

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by AceMkr**
### Description

Closes #1610 
Created an entry for the math function ".min()" in C#. I referenced other entries in math-functions/terms directory and tried to format my entry to match. The only exception is the addition of a "runnable codebyte example" as requested.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/min/min.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+# .Min()
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/min/min.md:15]: Remove this, we don't place h1 headers in entries, the metadata title will do this for us.

File: content/c-sharp/concepts/math-functions/terms/min/min.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+```
+
+This example results in the following output:
+
+```shell
+Math.Min (int) = -134
+```
+
+Now it's your turn, try changing the values of xint1 and xint2 to see the results.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/min/min.md:49]:
```suggestion
## Codebyte Example

The following code is runnable and the values of xint1 and xint2 can be changed to see the results.
```

We generally avoid first and second person pronouns, and the codebyte example should have a header.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@AceMkr, I did a first review and left a couple of comments,

File: content/c-sharp/concepts/math-functions/terms/min/min.md (hunk 1 above)
Comment (AceMkr) [reply content/c-sharp/concepts/math-functions/terms/min/min.md:15]: header removed

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by AceMkr** [comment]
> @AceMkr Congrats on the new entry. The live version can be seen here: https://www.codecademy.com/resources/docs/c-sharp/math-functions/min
> 
> It looks like you're anonymous: image
> 
> If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> To add your GitHub username to your Codecademy profile: 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box: image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login: image

My account is already linked as in step 1, but when I sign out and login with github as in step 2, it creates a new account. That new account that doesn't have any of my repositories or codecademy certificates is now showing as the contributor receiving credit. is there any way to fix this so that i can receive credit? also the codebyte sample i wrote is not functioning properly. Despite it running fine on my machine, the build fails on the docs page. I am attempting to fix the code now, should I create a new issue for this? or is there a way to reopen this pull request?

**Comment by SSwiniarski** [comment]
Any of our other maintainers: @yangc95, @KTom101, @caupolicandiaz, @HishamT have any idea on fixing @AceMkr's problem?

@AceMkr, you can submit a new PR to fix the codebyte.

**Comment by KTom101** [comment]
@AceMkr Are you able to check and see if the email you used for your Codecademy account is the email you have in your GitHub settings?

**Comment by AceMkr** [comment]
> @AceMkr Are you able to check and see if the email you used for your Codecademy account is an email you have in your GitHub settings?

That may be part of the issue. My Codecademy and Github were made with different email addresses. I'm not sure if I can change that without making a new account for either, but I will look into that. I'm wondering if there is a way to work around that... Thank you for the prompt reply!

**Comment by KTom101** [comment]
> > @AceMkr Are you able to check and see if the email you used for your Codecademy account is an email you have in your GitHub settings?
> 
> That may be part of the issue. My Codecademy and Github were made with different email addresses. I'm not sure if I can change that without making a new account for either, but I will look into that. I'm wondering if there is a way to work around that... Thank you for the prompt reply!

When I ran into this issue when linking my accounts, I added the email I used for my Codecademy profile as a primary email under my GitHub settings and kept the original email listed for making pull requests and notifications. You can have more than one email associated with your GitHub account. You can manage this under Settings -> Emails:

Screen Shot 2023-03-02 at 2 54 19 PM

Screen Shot 2023-03-02 at 2 55 03 PM

**Comment by AceMkr** [comment]
@KTom101 That fixed it! thank you so much for your prompt and thorough reply!

**Comment by KTom101** [comment]
@AceMkr You're Welcome!😁 Feel free to work on another [issue](https://github.com/Codecademy/docs/issues).

**Comment by yangc95** [comment]
🙌 When you do get a change to made a PR for edits, please refer to this original PR, thanks!

**Comment by yangc95** [comment]
@AceMkr Can we make an additional PR for this entry for some edits? This will be considered an enhancement.

It looks like the code byte is failing. Can we revisit the example and adjust accordingly.

> I can also assist with this change. Let me know how you would like to proceed.

**Comment by AceMkr** [comment]
Yes of course, I will work on it this week. I'm still very new to this but I believe I left out some pieces to the codebyte section that may have broken it. Thanks again for your help!

-Jeremy

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Pull request merged**